release_notes/topics/17_0_0.adoc

```diff
@@ -1,6 +1,20 @@
 = Highlights
 
-== Improved User Search
+== Quarkus distribution is now fully supported
+
+The default Keycloak distribution is now based on Quarkus. The new distribution is faster, leaner, and a lot easier to configure!
+
+We appreciate migrating from the WildFly distribution is not going to be straightforward for everyone, since how you start and configure Keycloak has radically changed. With that in mind we will continue to support the WildFly distribution until June 2022.
```

Comment:
```suggestion
We appreciate migrating from the WildFly distribution is not going to be straightforward for everyone, since how you start and configure Keycloak has radically changed. With that in mind, we will continue to support the WildFly distribution until June 2022.
```